src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -248,17 +248,19 @@ private static void BindProperty(PropertyInfo property, object instance, IConfig
                 return;
             }
 
+            object? propertyValue = property.GetValue(instance);
```

Comment:
what will happen if we just call `GetPropertyValue(property, instance, config, options);` here instead?

Reply:
I think more work needs to be done to investigate the correct fix that addresses https://github.com/dotnet/runtime/issues/52905.